### ISSUE TITLE
Changing firebase credentials

### ISSUE THREAD

**Issue by thayub** [duytq94/flutter-chat-demo#70]
Hi , the demo works out of the box perfectly, but when we change the google plist in iOS it crashes on clicking sign on button. 

Is there any specific thing to do or seed the database on my firebase account initially to start using the app ?

**Comment by Damscozy** [comment]
Yes you should fetch your firebase file which are google-services.json and GoogleServices-info.plist
